### ISSUE TITLE
Move deadline for a large amount of students

### ISSUE THREAD

**Issue by BushraRekabi** [devilry/devilry-django#1312]
Most courses allow students to use "Egenmelding" (self-certification) in order to get a postponed deadline (three days) if they can't deliver in time (because of sickness etc.). To achieve this, an administrator has to manually move the student's deadline and add a comment in Devilry. This is fine for a small number of students, but for some courses and assignments the number of deadlines that need to be moved has been as large as 300. This takes a while, because we have to search for the students and select them one by one under "Manage deadlines" --> "Move deadlines for some groups". It would help a lot if an administrator/examiner could be able to do this in a more efficient way (for example, given a list of student usernames - move the deadline for all), or if the students could manage this themselves in some way. 

Thank you!

**Comment by torgeirl** [comment]
Hi @BushraRekabi, and thank you for the suggestion. I do however believe we want to address use-cases like this (large courses, with the need of streamlining deadline extensions) more «in-system», possible with an opt-in option for all assignments on a course semester to handle deadline extensions: #1247 It is hard to give a timeline for when it will be implemented and unfortunately there are a lot more pressing issues that will need to be handled before we can get to it.
